pyproject.toml

```diff
@@ -16,6 +16,7 @@ dependencies = [
     "ruamel.yaml >=0.11.14,<0.19",
     "pillow >=3.1 ; platform_system=='Windows' or platform_system=='Darwin'",
     "jinja2",
+    "pydantic >=2"
```

Comment:
i've seen `pydantic` (and even `msgspec`) as a runtime dependency cause a great deal of pain, as there have been a number of breaking changes in the 2.x line, to say nothing of 1.x to 2.x. pydantic also makes icky, semi-unreproducible schema that almost always requires hand editing.

using `pydantic`, `msgspec`, etc. in an offline build process, checking in the JSON and/or generated plain-old-types (e.g. `TypedDict`), and using the JSON at runtime with e.g. `python-fastjsonschema` provides a much smaller surface area for failure.

Reply:
Yep, thanks for the reminder. I plan to do it like that eventually (as we did with conda-smithy), but for now as we develop it I'll keep it simple. Then we'll set up the CI machinery for the schema regeneration and so on 💪

Reply:
Done.